app/boards/blackpill_f401cc.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_CONSOLE=n
+CONFIG_SERIAL=n
+CONFIG_UART_CONSOLE=n
+CONFIG_UART_INTERRUPT_DRIVEN=n
+CONFIG_ZMK_USB=y
+CONFIG_ZMK_BLE=n
```

Comment:
This is probably superfluous? If preset in the others I would suggest removing them.